developers/weaviate/client-libraries/python/index.md

```diff
@@ -325,13 +325,14 @@ We recommend using the collection object to perform batch imports of single coll
 
 ### Batch sizing
 
-There are three methods to configure the batching behavior. They are `dynamic`, `fixed_size` and `rate_limit`.
+There are three methods to configure the batching behavior. They are `fixed_size`, `rate_limit` and `dynamic`.
 
 | Method | Description | When to use |
 | :-- | :-- | :-- |
-| `dynamic` | The batch size and the number of concurrent requests are dynamically adjusted on-the-fly during import, depending on the server load. | Recommended starting point. |
-| `fixed_size` | The batch size and number of concurrent requests are fixed to sizes specified by the user. | When you want to specify fixed parameters. |
+| `fixed_size` | The batch size and number of concurrent requests are fixed to sizes specified by the user. | When you want to specify fixed parameters. Recommended method. |
 | `rate_limit` | The number of objects sent to Weaviate is rate limited (specified as n_objects per minute). | When you want to avoid hitting third-party vectorization API rate limits. |
+| `dynamic` | The batch size and the number of concurrent requests are dynamically adjusted on-the-fly during import, depending on the server load. | Not recommended for production environments at this time. |
```

Comment:
I think the limitation is specific to WCD serverless instances, because the dynamic batcher can trigger scaling events. So perhaps we could say `Not recommended for Weaviate Cloud Serverless instances at this time`. 

Afaik in other prod environments it may be okay.